### DISCUSSION TITLE
Train single speaker to multi-speaker by adding a new speaker

### DISCUSSION THREAD

**Discussion by kin0303** [coqui-ai/TTS#2094] [General Q&A]
I want to train using a single speaker, and when the output sounds good, I want to add a new speaker. Can I do that? And the model will be multi-speakers model

**Comment by erogol** [comment]
Yes you can do that. You can either expand the speaker embedding layer or just add the new speaker d-vectors and  continue training.

**Reply by kin0303** [reply]
> just add the new speaker d-vectors and continue training.

How's that? Can I do that by enabling use_speaker_embedding?

**Reply by huongtxvega** [reply]
Hi @kin0303, Have you tried it yet? Can you say more about this.
